D3/index.html

```diff
@@ -1,87 +1,118 @@
 <html>
+  <head>
+    <link rel="stylesheet" href="node_modules/reveal.js/css/reveal.css" />
+    <link rel="stylesheet" href="node_modules/reveal.js/css/theme/white.css" />
+    <link rel="stylesheet" href="node_modules/reveal.js/lib/css/zenburn.css" />
 
-    <head>
-        <link rel="stylesheet" href="node_modules/reveal.js/css/reveal.css">
-        <link rel="stylesheet" href="node_modules/reveal.js/css/theme/white.css">
-        <link rel="stylesheet" href="node_modules/reveal.js/lib/css/zenburn.css">
+    <style>
+      .reveal .slides section .source {
+        font-size: 0.3em;
+      }
 
-        <style>
-            .reveal .slides section .source {
-                font-size: 0.3em;
-            }
+      .reveal .slides section .code-file-name {
+        width: 85%;
+        margin: 0 auto -0.5em auto;
+        text-align: left;
+        font-size: 0.7em;
+      }
+    </style>
 
-            .reveal .slides section .code-file-name {
-                width: 85%;
-                margin: 0 auto -0.5em auto;
-                text-align: left;
-                font-size: 0.7em;
-            }
-        </style>
+    <script>
+      var link = document.createElement('link');
+      link.rel = 'stylesheet';
+      link.type = 'text/css';
+      link.href = window.location.search.match(/print-pdf/gi)
+        ? 'node_modules/reveal.js/css/print/pdf.css'
+        : 'node_modules/reveal.js/css/print/paper.css';
+      document.getElementsByTagName('head')[0].appendChild(link);
+    </script>
+  </head>
 
-        <script>
-            var link = document.createElement('link');
-            link.rel = 'stylesheet';
-            link.type = 'text/css';
-            link.href = window.location.search.match(/print-pdf/gi) ? 'node_modules/reveal.js/css/print/pdf.css' : 'node_modules/reveal.js/css/print/paper.css';
-            document.getElementsByTagName('head')[0].appendChild(link);
-        </script>
-    </head>
+  <body>
+    <div class="reveal">
+      <div class="slides">
+        <section
+          data-markdown="materials/Week 1/Slides/slides.md"
```

Comment:
You can remove week 1 slides.md from here since they're in the design-series folder.

Reply:
done